restapi/src/main/java/io/stargate/web/resources/v2/RowsResource.java

```diff
@@ -602,8 +610,13 @@ private Response modifyRow(
           .build();
     }
 
-    @SuppressWarnings("unchecked")
-    Map<String, Object> requestBody = mapper.readValue(payload, Map.class);
+    Map<String, Object> requestBody;
+    try {
```

Comment:
... since this is an idiom used in a few places, might want to just extract into separate method that handles error catch/rethrow?

Reply:
Sure.